### PR TITLE
[CMake][NFC] Added test case for keyword in CMake target

### PR DESCRIPTION
**Pull request by TejaX-Alaghari**
Added a new case for target_link_libraries to test keyword in CMake target

### CODE REVIEW

File: clang/test/dpct/cmake_migration/case_004/input.cmake [hunk 1]
```diff
@@ -19,5 +19,7 @@ target_link_libraries(${PROJECT_NAME}
    libnvinfer.so
    libnvonnxparser.so
 )
+
 target_link_libraries(nlm_cuda  ${OpenCV_LIBS} stdc++ stdc++fs)
+
```
Comment (zhimingwang36) [inline clang/test/dpct/cmake_migration/case_004/input.cmake:24]: this is NFC change?

Comment (TejaX-Alaghari) [reply clang/test/dpct/cmake_migration/case_004/input.cmake:24]: Yeah Zhiming

**Comment by zhimingwang36** [comment]
need rebase this PR to solve the conflict.

**Comment by TejaX-Alaghari** [comment]
This change was already added by a different PR.

Hence, closing this PR.

**Pull request closed without merging**